ext/TrixiNLsolveExt.jl

```diff
@@ -0,0 +1,79 @@
+# Package extension for adding Convex-based features to Trixi.jl
+module TrixiNLsolveExt
+
+# Required for coefficient optimization in P-ERK scheme integrators
+if isdefined(Base, :get_extension)
+    using NLsolve: nlsolve
+else
+    # Until Julia v1.9 is the minimum required version for Trixi.jl, we still support Requires.jl
+    using ..NLsolve: nlsolve
+end
+
+# We use a random initialization of the nonlinear solver.
+# To make the tests reproducible, we need to seed the RNG
+using Random: seed!
```

Comment:
This is not guaranteed to work across different Julia versions. If you want to ensure reproducible results, you have to avoid randomness - or use something like https://github.com/JuliaRandom/StableRNGs.jl

Reply:
Got it! Thank you!